content/docs/12.5-vertex/60-system-management/50-reverse-proxy.md

```diff
@@ -30,7 +29,7 @@ You can deploy and maintain your own Spectro Proxy server. The Spectro Proxy ser
 - [Helm](https://helm.sh/docs/intro/install/) is installed and available.
 
 
-- Access to the target Kubernetes cluster's kubeconfig file. You must be able to interact with the cluster using `kubectl` commands and have sufficient permissions to install Palette VerteX. We recommend using a role with cluster-admin permissions to install Palette VerteX.
+- Access to the Palette VerteX Kubernetes cluster's kubeconfig file. You can download the kubeconfig file from the Palette VerteX system console. Navigate to **Enterprise System Migration**, select the Palette VerteX cluster, and click the **Download Kubeconfig** button for the cluster.
```

Comment:
```suggestion
- Access to the kubeconfig file of the Palette VerteX Kubernetes cluster. You can download the kubeconfig file from the Palette VerteX system console. Navigate to **Enterprise System Migration**, select the Palette VerteX cluster, and click the **Download Kubeconfig** button for the cluster.
```

Reply:
FYI, the reason for this suggestion is that when appostrophe 's' is used with the plural in the same sentence, it's hard to read. You can hear it if you read it aloud.